db/db-migration.py

```diff
@@ -0,0 +1,12 @@
+from yoyo import read_migrations, get_backend
+
+# Change user and host as per requirement
+user = "aadarsh"
+host = "localhost"
```

Comment:
you should get it from the local_setting.py (one should create).

```suggestion
from local_settings import user, host

```

@CharlesNepote do you agree ?

Reply:
@aadarsh-ram it's up to the person installing the program to rename it. It should be documented in installation I think.

I would:
* keep local_settings_example.py in repo
* add a note in the installation doc to tell to copy this file as local_settings.py
* add local_settings.py to gitignore.

Reply:
@alexgarel I'll go ahead and implement this. I'll also replace the current API code with the already existing one as discussed [here](https://github.com/openfoodfacts/folksonomy_api/issues/49#issuecomment-1088489885). Does this sound good?